packages/text-editor/src/provider/minio.ts

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright © 2023 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import { getMetadata } from '@hcengineering/platform'
+import presentation from '@hcengineering/presentation'
+import { concatLink } from '@hcengineering/core'
+import { ObservableV2 as Observable } from 'lib0/observable'
+import { type Doc as YDoc, applyUpdate } from 'yjs'
+
+interface EVENTS {
+  synced: (...args: any[]) => void
+}
+
+async function fetchContent (doc: YDoc, name: string): Promise<void> {
+  const frontUrl = getMetadata(presentation.metadata.FrontUrl) ?? window.location.origin
+  const token = getMetadata(presentation.metadata.Token) ?? ''
+
+  const res = await fetch(concatLink(frontUrl, `/files?file=${name}&token=${token}`))
```

Comment:
As I remember we use cookie, do we really need to pass token?

Reply:
Likely no, let me remove it ...